src/App.css

```
@@ -0,0 +1,14 @@
+.or {
+  padding: 34px 145px 0 145px;
+  font-weight: 300;
+  font-size: 18px;
+  line-height: 0px;
+  text-align: center;
+  color: rgba(77, 71, 139, 0.5);
+}
+
+.background-style {
+  background-image: url(./fondohome.png);
+  display: grid;
+  justify-items: center;
+}
```

Beautiful image!

Considero el nombre de la clase de CSS `background-style`, es un poco confuso con las propiedades de CSS, es decir: no solo se esta modificando el background sino también el layout. Por la ubicación donde se esta usando, considero renombrar el nombre a : `.App`, `.home`, `.app-content`, etc.

```suggestion
.App {
  background-image: url(./fondohome.png);
  display: grid;
  justify-items: center;
}
```

¿Qué opinan ustedes al respecto?

---

De acuerdo, ahora se llama `.app-content`.

---

listo!